server/hotrod/src/main/resources/META-INF/services/org.infinispan.factories.components.ModuleMetadataFileFinder

```diff
@@ -0,0 +1 @@
+org.infinispan.server.hotrod.ServerMetadataFileFinder
```

Comment:
The services generation stuff is a javac annotation processor, that gets simple ignored in the ```server/hotrod``` module since it's compiled by scalac. Not sure if there's any workaround available, will leave as for now